pkg/application/cluster_app.go

```diff
@@ -46,22 +46,6 @@ func NewClusterApp(clusterName string, provider Provider) *Cluster {
 	}
 }
 
-// WithOrg sets the Organization for the cluster and updates the namespace to that specified by the provided Org
-func (c *Cluster) WithOrg(org *organization.Org) *Cluster {
```

Comment:
Did you mean to remove this? It's now not possible to manually set an org which would prevent any tests that create two clusters in the same org.

Reply:
I added it back, please take a look.